api/src/main/java/io/strimzi/api/kafka/model/rebalance/KafkaRebalanceStatus.java

```diff
@@ -25,11 +25,12 @@
         builderPackage = Constants.FABRIC8_KUBERNETES_API
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({ "conditions", "observedGeneration", "sessionId", "optimizationResult" })
+@JsonPropertyOrder({ "conditions", "observedGeneration", "sessionId", "progress", "optimizationResult" })
 @EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
 public class KafkaRebalanceStatus extends Status {
     private String sessionId;
+    private Map<String, Object> progress = new HashMap<>(0);
```

Comment:
Right now we have just a "rebalanceProgressConfigMap" field. Is the plan to have freedom to add more fields in the future? For this reason this is a map and not a strongly typed field?

Reply:
>  Is the plan to have freedom to add more fields in the future? For this reason this is a map and not a strongly typed field?

That was the initial idea, just in case we want to add any fields here in the future.